VoodooPS2Controller/VoodooPS2Controller.cpp

```diff
@@ -489,7 +489,10 @@ bool ApplePS2Controller::start(IOService * provider)
   // Reset and clean the 8042 keyboard/mouse controller.
   //
     
-  resetController();
+  PE_parse_boot_argn("ps2rst", &_resetControllerFlag, sizeof(_resetControllerFlag));
+  if (_resetControllerFlag & RESET_CONTROLLER_ON_BOOT) {
```

Comment:
I think we do not usually add braces around singlle-line conditionals, mind removing them? I would have ignored this, but two different styles are used anyway here and in setPowerStateGated.

Reply:
I just attempted to match the style in the context. The "if" blocks in setPowerStateGated are with braces in new lines.

I can remove the braces, but I notice some codes after "if" are indented with four spaces, some others are indented with two. What do you think to be better consistent?

Reply:
Ugh, that's really bad. In general it should be 4, but some files are indeed messed up. I guess it is not right to ask you to fix it. Will merge as is for now. Thanks!

Reply:
OK. You are welcome.